e2e/pages/events/messageJudgeOrLegalAdviserEvent.page.js

```diff
@@ -21,16 +21,20 @@ module.exports = {
       },
     },
     closeMessageLabel: '#judicialMessageReply_closeMessageLabel',
-    requested: '#judicialMessageMetaData_requestedBy',
+    subject: '#judicialMessageMetaData_requestedBy',
```

Comment:
Does the selector need to be `judicialMessageMetaData_subject`?

Reply:
it would require data migration